pkg/report/report.go

```diff
@@ -50,8 +51,12 @@ type Report struct {
 	Corrupted bool
 	// CorruptedReason contains reason why the report is marked as corrupted.
 	CorruptedReason string
-	// Maintainers is list of maintainer emails (filled in by Symbolize).
+	// Maintainers is a list of maintainer emails (filled in by Symbolize).
 	Maintainers []string
+	// Cc is a list of less important emails (filled in by Symbolize).
+	Cc []string
```

Comment:
Is this used?
I would assume Recipients should eliminate the need in both CC and Maintainers.